src/@carbon-info/views/RoadMapPage/RoadMapPage.tsx

```diff
@@ -20,23 +21,37 @@ const RoadMapPage: React.FC = () => {
       const content = await data as any;
       if (content && Array.isArray(content.items)) {
         content.items.forEach((o: any) => {
-          let tabs: any[] = [];
+          let inProgress: any[] = [];
+          let completed: any[] = [];
           o?.fields?.entries?.forEach((entry: any) => {
-            tabs.push({
+            let data = {
               docLink: entry.fields.link,
               githubLink: entry.fields.githubLink,
               progress: entry.fields.progress,
               status: entry.fields.status,
               title: entry.fields.title,
               description: entry.fields.description.content[0].content[0].value,
               shortDescription: entry.fields.shortDescription,
-            });
+            };
+            if (entry.fields.progress === 100) {
+              completed.push({ ...data });
+            } else {
+              inProgress.push({ ...data });
+            }
           });
 
+          inProgress.sort((a, b) => (b.progress - a.progress));
+          completed.sort(((a, b) => {
+            if (moment(a.lastUpdate) > moment(b.lastUpdate)) {
+              return 1;
+            }
+            return -1;
+          }));
```

Comment:
Think to make this algorithm more efficient, you can use 1 map function and 1 sort function:
```typescript
const tabs = o?.fields?.entries.map((entry: any) => ({
  docLink: entry.fields.link,
  githubLink: entry.fields.githubLink,
  ...
})).sort((a, b) => {
  if (a.progress === 100 && b.progress === 100) {
    return moment(a.lastUpdate) > moment(b.lastUpdate) ? 1 : -1
  }
  ...
})
result.push({ ..., tabs })
```